x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -72,23 +71,23 @@ export function App({
     indexPatternsForTopNav: [],
     query: { query: '', language },
     dateRange: {
-      fromDate: timeDefaults.from,
-      toDate: timeDefaults.to,
+      fromDate: data.query.timefilter.timefilter.getTime().from,
```

Comment:
Seems like `data.query.timefilter.timefilter.getTime()` should be pulled out into a local const, since that's a mouthful and it's repeated twice.